pkg/sbom/io/encode_test.go

```diff
@@ -0,0 +1,364 @@
+package io_test
```

Comment:
is this test should be removed ?

Reply:
I'll update the test.

Reply:
Added https://github.com/aquasecurity/trivy/pull/6240/commits/dda7d7dd4fcb6cc5f2844906c28bda75d6a02e7c